src/index.tsx

```diff
@@ -25,6 +25,10 @@ const problem = curriculumUnit.getProblem(problemOrdinal) ||
 
 const stores = createStores({ appMode, user, problem });
 
+if (problem) {
+  document.title = `CLUE: ${problem.getFullTitle()}`;
```

Comment:
For the browser tab we might just want the title rather than the full title.

Reply:
I was thinking the full title so they know what problem they are on.